core/src/com/mygdx/game/util/Const.java

```diff
@@ -0,0 +1,10 @@
+package com.mygdx.game.util;
```

Comment:
Nazwa bardzo nijaka, GameConst? No i po co ten konstruktor?

Reply:
konstruktor jest po to, żeby nie było możliwości instancjonowania tej klasy, bo służy tylko do Constów, zapomniałem zrobić ją finalną

a nazwa GameConst jest tak samo nijaka jak Const

mogę w sumie to przenieść do Assets, albo obok Assets jako AssetPaths.java i wtedy będzie lepiej